content/applications/general/email_communication/email_servers.rst

```diff
@@ -2,221 +2,208 @@
 Send and Receive Emails in Odoo with an Email Server
```

Comment:
Replace entire header with `Send and receive emails in Odoo with an email server`